File: C++/Insertion.cpp
#include <iostream>
#include <conio.h>
using namespace std;
int main()
{
    int a[10], i, n, j, k, value;
    cout << "Enter number of elements to be entered: ";
    cin >> n;
    for (i = 0; i < n; i++)
    {
        cin >> a[i];
    }
    cout << "Enter at what location do you want to insert the new value: ";
    cin >> k;
    cout << "Enter the value: ";
    cin >> value;
    // Insertion part
    i = n;
    while (i != k)
    {
        a[i] = a[i - 1];

        i--;
    }
    a[k] = value;
    for (i = 0; i <= n; i++)
    {
        cout << a[i] << ' ';
    }
}

File: Insertion.py
n=int(input('Enter number of elements to be entered: '))
a=[]
for i in range(n):
    a.append(int(input()))
index=int(input('Enter the location: '))
value=int(input('Enter the value: '))
a.append(0)
for j in range(n,index,-1):
    a[j]=a[j-1]
a[index]=value
print(a)